src/scripts/eastern_kingdoms/dun_morogh/gnomeregan/gnomeregan.cpp

```diff
@@ -35,33 +35,33 @@ EndContentData */
 
 enum
 {
-    SAY_START                   = -1090000,
-    SAY_INTRO_1                 = -1090001,
-    SAY_INTRO_2                 = -1090002,
```

Comment:
Script text -1090002 has emote = 5 (Exclamation), while broadcast text id 4052 which u replace it with currently has no emote, but you do not update the broadcast text in your migration.